yapapi/props/base.py

```diff
@@ -57,13 +58,13 @@ def get_type_args(t):
             else:
                 return getattr(t, "__args__")
 
-        # print("name=", self.name, "type=", self.type, type(self.type))
+        # print("name=", self.name, "type=", self.type, type(self.type), "value=", value)
         if get_type_origin(self.type) == Union:
             if datetime in get_type_args(self.type):
                 # TODO: fix this.
                 return self.name, datetime.fromtimestamp(int(float(value) * 0.001), timezone.utc)
             return self.name, value
-        if issubclass(self.type, enum.Enum):
+        if inspect.isclass(self.type) and issubclass(self.type, enum.Enum):
```

Comment:
Why is `issubclass(self.type, enum.Enum)` not enough? Could this check return `True` (or fail with an error) for a non-class type?

Reply:
yes, it fails when `self.type` is not a class

Reply:
Are you sure? In my Python interpreter it is correctly evaluated:
```
>>> issubclass(type(5), enum.Enum)
False
>>> issubclass(str, enum.Enum)
False
```